interface/src/app/gene/gene-network.tpl.html

```diff
@@ -3,8 +3,19 @@ <h1 class="page-header">
     Gene Network
   </h1>
   <div class="well">
-    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
I always prefer modeless over modal, if possible. This reduces clicks and makes the features more readily accessible to the user. Sometimes modal "dialog boxes" are unavoidable, but in most cases you can design a non-modal way of doing it that will be better.

In this particular case, what if the page has a few scattered terms the user wants to copy and paste into the search box? They can't do that if the input field is inaccessible in a modal.